pypolychord/polychord.py

```diff
@@ -365,3 +786,7 @@ def write(var):
             write('---------------------------------------')
             write('=== dead equally weighted posterior points ===')
             write('=== global equally weighted posterior points ===')
+
+def _legacy_make_resume_file(settings, loglikelihood, prior):
+    kwargs = settings.__dict__
+    _make_resume_file(loglikelihood, prior = prior, **kwargs)
```

Comment:
I realised I could dispense with the enormous duplication between `run` and the old `run_polychord` with this same approach. @williamjameshandley what do you think?

Reply:
@williamjameshandley could do with looking at this before I completely forget what I was thinking